project-fob/Controllers/HomeController.cs

```diff
@@ -132,9 +128,17 @@ public ActionResult MeetingPageUser(string meetingId, string password)
             return View("Index");
         }
 
+        public static string GenerateGuid()
+        {
+            Guid g;
+            g = Guid.NewGuid();
+
+            return g.ToString("N");
+        }
+
         public static string GenerateId()
         {
-            return GenerateIdByLength(9);
+            return GenerateGuid();
```

Comment:
Given that all this method does is call `GenerateGuid()`, I'd be tempted to just inline that method (i.e. put the code from `GenerateGuid` into this method and delete `GenerateGuid`).